From c485dde34ad08aa213d9948a40292958cc027cd0 Mon Sep 17 00:00:00 2001
From: Vinitha V Pillai <vinitha.pillai@nxp.com>
Date: Tue, 17 Jul 2018 10:34:09 +0530
Subject: [PATCH 068/107] armv8: LS1088ARDB: Fix QUBOOT-3935

Enable PCIe config options in LS1088 SD Secure Boot defconfig

Signed-off-by: Vinitha V Pillai <vinitha.pillai@nxp.com>
---
 .../ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig   |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig b/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig
index 7341fef..974b8fb 100644
--- a/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig
+++ b/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig
@@ -43,6 +43,10 @@ CONFIG_FSL_SPI_ALIGNED_TXFIFO=y
 CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_NETDEVICES=y
 CONFIG_E1000=y
+CONFIG_PCI=y
+CONFIG_DM_PCI=y
+CONFIG_DM_PCI_COMPAT=y
+CONFIG_PCIE_LAYERSCAPE=y
 CONFIG_SYS_NS16550=y
 CONFIG_DM_SPI=y
 CONFIG_FSL_DSPI=y
-- 
1.7.1

